app/policies/grant_policy.rb

```diff
@@ -23,31 +19,34 @@ def show?
     when true
       user.present?
     else
-      organization_admin_access? || grant_viewer_access?
+      grant_viewer_access?
```

Comment:
How soon is this coming out? I don't like how this works differently than the grant_editor_policy.

Reply:
We can open a ticket and add it to the next sprint.